UserService/UserService.Core/Services/TrackLoginService.cs

```diff
@@ -40,8 +51,114 @@ public async Task CheckLogins()
 
             await _repository.UpdateLoginChecks(dtChecked, history);
 
-            int i = 1;
+            await ManageInactiveUsers(2);
+        }
+
+        public async Task ManageInactiveUsers(int yearsInActive)
```

Comment:
This method could be shortened by adding one or two private methods for sending emails.